.github/workflows/ci.yaml

```
@@ -0,0 +1,143 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: IN-CORE/incore-docs
+
+jobs:
+
+  # ----------------------------------------------------------------------
+  # DOCKER BUILD
+  # ----------------------------------------------------------------------
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+    # checkout source code
+    - uses: actions/checkout@v2
+
+   # calculate some variables that are used later
+    - name: version information
+      run: |
+        if [ "${{ github.event.release.target_commitish }}" != "" ]; then
+          BRANCH="${{ github.event.release.target_commitish }}"
+        elif [[ $GITHUB_REF =~ pull ]]; then
```

This condition can be optimized as below so it doesn't consider the branch names with word "pull" in it
```suggestion
        elif [[ "${{github.event_name}}" == "pull_request" ]]; then
            BRANCH="PR-${{github.event.pull_request.number}}"
```